yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -289,6 +269,22 @@ func (p *ParserIssueDetector) getDDLIssues(query string) ([]QueryIssue, error) {
 			issues[i].SqlStatement = query
 		}
 	}
+
+	if _, ok := ddlObj.(*queryparser.Object); ok { // In case the DDL doesn't have any processor skip checking generic issues
+		return issues, nil
+	}
+
+	genericIssues, err := p.genericIssues(query)
```

Comment:
Add a comment here explaining why we're making this call for DDLs. Give an example.